django/contrib/gis/gdal/feature.py

```diff
@@ -82,7 +82,8 @@ def num_fields(self):
     @property
     def fields(self):
         "Return a list of fields in the Feature."
-        return [capi.get_field_name(capi.get_field_defn(self._layer._ldefn, i))
+        return [force_text(capi.get_field_name(capi.get_field_defn(self._layer._ldefn, i)),
```

Comment:
I'd suggest this for the indentation:
```python
return [
    force_text(
        capi.get_field_name(capi.get_field_defn(self._layer._ldefn, i)),
        self.encoding,
        strings_only=True,
    ) for i in range(self.num_fields)
]
```
Any concern about having no tests for the `self.encoding, strings_only=True` bit? I'm not sure we would expect non-string values in the first place? Also, at least on my system, there are no failures with it removed here or in `layer_name` above.